TobiiTest/Preference.xaml.cs

```diff
@@ -65,6 +70,10 @@ private void InitComboBoxes()
             translatorCB.Items.Clear();
             translatorCB.Items.Add("Google");
             translatorCB.Items.Add("Microsoft");
+            setKey.Items.Clear();
+            setKey.Items.Add("Left Ctrl");
+            setKey.Items.Add("Left Shift");
+            setKey.Items.Add("Left Ctrl+Alt");

```

Comment:
Ctrl+Alt will be hard to detect. Better stick with just one key, not key combinations. You can add right ctrl and shift, for example.

Reply:
I have changed to "Right Shift" and "Right Ctrl"